affiliation/handlers.go

```diff
@@ -976,6 +1067,9 @@ func Configure(api *operations.DevAnalyticsAffiliationAPI, service Service) {
 				"X-REQUEST-ID": requestID,
 			}).Info("DeleteEnrollmentsHandlerFunc: " + info)
 
+			if service.IsProjectSkipped(params.ProjectSlugs) {
```

Comment:
`ProjectSlugs` can be a `,` separated list of project slugs.

Reply:
didnt know this, added support for this.